ledger/complete/wal/checkpoint_v6_reader.go

```diff
@@ -78,15 +78,18 @@ func ReadCheckpointV6(headerFile *os.File, logger *zerolog.Logger) ([]*trie.MTri
 }
 
 // OpenAndReadCheckpointV6 open the checkpoint file and read it with ReadCheckpointV6
-func OpenAndReadCheckpointV6(dir string, fileName string, logger *zerolog.Logger) ([]*trie.MTrie, error) {
+func OpenAndReadCheckpointV6(dir string, fileName string, logger *zerolog.Logger) (
+	tries []*trie.MTrie,
+	errToReturn error,
```

Comment:
if you are thinking why not just `err`, then please read the comments on `closeAndMergeError`